src/render.js

```diff
@@ -5,6 +5,22 @@ export const renderPosition = {
   AFTEREND: 'afterend',
 };
 
-export const renderTemplate = (container, position, template) => {
-  container.insertAdjacentHTML(position, template);
+export const render = (container, place, element) => {
+  switch (place) {
+    case renderPosition.BEFOREBEGIN:
+      return container.before(element);
+    case renderPosition.AFTERBEGIN:
+      return container.prepend(element);
+    case renderPosition.BEFOREEND:
+      return container.append(element);
+    case renderPosition.AFTEREND:
+      return container.after(element);
+  }
+};
+
+export const createElement = (template) => {
+  const newElement = document.createElement('div');
+  newElement.innerHTML = template;
+
+  return newElement.firstChild;
```

Comment:
Так лучше, разницу знаешь ?
```suggestion
  return newElement.firstElementChild;
```

Reply:
в том что firstElementChild - возвращает элемент, игнорируя текст и комменты если например они идут перед элементом. 
firstChild - может вернуть элемент, текст или комменты в зависимости от того какой первый